introduction.md

```diff
@@ -0,0 +1 @@
+hello, but it looks to be empty, or just too short. We really want to know something about you! Make sure to include a short sentance about yourself
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.